src/domogik/tests/common/helpers.py

```diff
@@ -96,18 +96,16 @@ def delete_configuration(type, name, host):
     msg.add_data('type', type)
     msg.add_data('host', host)
     msg.add_data('name', name)
-    result = cli.request('admin', msg.get(), timeout=10)
+    result = cli.request('admin', msg.get(), timeout=20)
```

Comment:
At Some time admin was to much busy and don't respond in 10s. When we use Plugin Testcase,  we delete at each run plugin config and I notice increase this timeout perform this fonction. We could put 15s also.